src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,46 @@
+.stopwatch {
+  width: 2 * $length-second-hand;
+  height: 2 * $length-second-hand;
+  background-color: #00f;
+  border-radius: 50%;
+
+  @keyframes ticker {
+    100% {
+      transform: rotate(360deg);
+    }
+  }
+
+  &__second-hand {
+    position: absolute;
+    top: calc(50% - #{$length-second-hand});
+    left: calc(50% - 0.5 * #{$width-second-hand});
+    width: $width-second-hand;
+    height: $length-second-hand;
+    background-color: #f00;
+    border-radius: 50%;
+    transform-origin: bottom center;
+    animation: ticker 60s steps(60) 0.5s infinite;
+  }
+
+  &__minute-hand {
+    position: absolute;
+    top: calc(50% - #{$length-minute-hand});
+    left: calc(50% - 0.5 * #{$width-minute-hand});
+    width: $width-minute-hand;
+    height: $length-minute-hand;
+    background-color: #0f0;
+    border-radius: 50%;
+    transform-origin: bottom center;
+    animation: ticker 3600s steps(60) 0.5s infinite;
```

Comment:
u r reusing animation property, I think u can create a mixin in separate file which would look something like following
```
@mixin ticker ($period, $position) {
  animation: rotate #{$period} infinite linear;
  transform-origin: #{$position};

  @keyframes rotate {
    from {
      transform: rotate(0);
    }
    to {
      transform: rotate(360deg);
    }
  }
}
```
and just include the mixin instead of `animation` property in your classes

Reply:
Done. Moved animation to mixin. In the mixin there is only one parameter - period. 
All other parameters need to stay always the same.